pkg/pki/ssh/verify.go

```diff
@@ -0,0 +1,35 @@
+package ssh
+
+import (
+	"crypto/sha512"
+
+	"golang.org/x/crypto/ssh"
+)
+
+func Verify(message []byte, armoredSignature string, publicKey []byte) error {
```

Comment:
use `io.Reader` instead of `byte[]` in case it is a large payload

Reply:
Done.